bindata/v3.11.0/openshift-apiserver/defaultconfig.yaml

```diff
@@ -42,6 +42,11 @@ auditConfig:
           - "/api*" # Wildcard matching.
           - "/version"
           - "/healthz"
+      - level: RequestResponse
```

Comment:
We have an explanation for the reasoning behind the rules everywhere else. Let's add one here too.

Reply:
Sure, thanks for the review. Are you OK with the comment I added in the latest iteration?

Reply:
looks good

Reply:
looks good